tequilapi/endpoints/identities.go

```diff
@@ -29,10 +29,23 @@ import (
 	"github.com/mysterium/node/tequilapi/validation"
 )
 
+// Used for operations that want identity in the path
+// swagger:parameters registerIdentity
+type IdentityURLParams struct {
```

Comment:
Is it possible to re-use this struct inside code when parsing request params?

Reply:
I think it should be possible

Reply:
found way to declare parameters in other way, so we no longer need struct IdentityURLParams

Reply:
I agree, would be good to investigate parameters more. https://goswagger.io/generate/spec/params.html